src/service_router.rs

```diff
@@ -4,7 +4,7 @@
 
 use context::{ ContextTrait, Shared };
 use iron::{Request, Response, IronResult};
-use iron::headers::ContentType;
+use iron::headers::{ ContentType, AccessControlAllowOrigin };
```

Comment:
I added them to get my demo client working. I guess we'll change/revert once we have proper user authentication working.